### DISCUSSION TITLE
how do i update my containers example radarr

### DISCUSSION THREAD

**Discussion by vandieberg** [tteck/Proxmox#2229] [Help Needed] [answered]
how do i update my containers created from tteck scripts to the latest version?

**Comment by AlexGeek77** [comment, marked as answer]
Each script on the page gives information on how to upgrade. For most, it's as simple as running 'update' in the console of the container.

**Reply by vandieberg** [reply]
i take it it needs to be an unprivileged container to use the update option?

**Reply by tteck** [reply]
No

**Comment by tteck** [comment]
Or, if it doesn't specify on the website, it means that the update process is performed within the application itself or no update path has been resolved.  
In the case of Radarr, click on System in the left main navigation menu and select Updates.
